### PR TITLE
feat: dashboard of all alerts

### PR DESCRIPTION
**Pull request by ytvnr**
Fixes gravitee-io/issues#4892

image
image
image
image

### CODE REVIEW

File: src/components/alerts/dashboard/_alerts-dashboard.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
+.empty {
```
Comment (gcusnieux) [inline src/components/alerts/dashboard/_alerts-dashboard.scss:1]: This class will apply to the whole project. Please use BEM or scss structure to apply it only in this case.

File: src/management/dashboard/dashboard.html [hunk 2]
```diff
@@ -29,6 +29,10 @@
       <md-tab-label><ng-md-icon icon="insert_chart" style="fill: #28444F;"></ng-md-icon> Analytics</md-tab-label>
       <md-tab-body></md-tab-body>
     </md-tab>
+    <md-tab ng-if="$ctrl.alertsEnabled" ui-sref="management.dashboard.alerts" permission permission-only="'environment-alert-r'">
+      <md-tab-label><ng-md-icon icon="alarm" style="fill: #28444F;"></ng-md-icon> Alerts</md-tab-label>
```
Comment (gcusnieux) [inline src/management/dashboard/dashboard.html:33]: Why do not use gv-icon ? The goal is to remove md-icon soon as possible

File: src/services/alert.service.ts [hunk 3]
```diff
@@ -17,6 +17,20 @@
 import { IHttpPromise } from 'angular';
 import { Alert, Scope } from '../entities/alert';
 
+// tslint:disable-next-line:interface-name
```
Comment (gcusnieux) [inline src/services/alert.service.ts:20]: Why not apply this rule?

File: tsconfig.json [hunk 4]
```diff
@@ -16,7 +16,8 @@
     "types": [
       "jest",
       "angular-mocks"
-    ]
+    ],
+    "esModuleInterop": true
```
Comment (gcusnieux) [inline tsconfig.json:20]: Which module needs this?

**Review by gcusnieux — CHANGES_REQUESTED** [review]

File: src/management/dashboard/dashboard.html (hunk 2 above)
Comment (ytvnr) [reply src/management/dashboard/dashboard.html:33]: for the moment gv-icon do not have all the states required for "Alerts" pages (not only this menu)
as discussed, we will keep it as it is

File: src/components/alerts/dashboard/_alerts-dashboard.scss (hunk 1 above)
Comment (ytvnr) [reply src/components/alerts/dashboard/_alerts-dashboard.scss]: deleted because i don't need it anymore

File: src/services/alert.service.ts (hunk 3 above)
Comment (ytvnr) [reply src/services/alert.service.ts]: deleted and use interface prefixed with i

File: tsconfig.json (hunk 4 above)
Comment (ytvnr) [reply tsconfig.json]: i forget to remove this change :)

**Review by gcusnieux — APPROVED** [review]

**Pull request merged**